catalyst/utils/metrics/cmc_score.py

```diff
@@ -0,0 +1,50 @@
+import torch
+
+
+def cmc_score(
+    query_embeddings: torch.Tensor,
+    gallery_embeddings: torch.Tensor,
+    conformity_matrix: torch.Tensor,
+    topk: int = 1,
+) -> float:
+    """
+
+    Args:
+        query_embeddings: tensor shape of (n_embeddings, embedding_dim)
+            embeddings of the objects in querry
+        gallery_embeddings: tensor shape of (n_embeddings, embedding_dim)
+            embeddings of the objects in gallery
+        conformity_matrix: binary matrix with 1 on same label pos
```

Comment:
do we have plans for `-1` (aka hard negatives support)?

Reply:
Well, we (me and Aleksey) agreed that negative classes is very unusual case and probably we wont add negative classes support for now.

Reply:
@AlekseySh aren't they? from my perspective hard negatives are common thing to think about

Reply:
Agree with Nikita